README.md

```diff
@@ -31,3 +33,45 @@ prop      | type                 | default value
 `level`   | `string` (`'L' 'M' 'Q' 'H'`)            | `'L'`
 
 <img src="qrcode.png" height="256" width="256">
+
+
+## Downloading the *qr code* as an image
+
+The *qr code* generated by the component is a [canvas](http://www.w3schools.com/html/html5_canvas.asp). If you need to create a button to download the qr code image you can use the component's `ref` prop like this:
+
+```js
+class Component extends React.Component {
+  constructor(props) {
+    super(props);
+    this.getCanvasNode = this.getCanvasNode.bind(this);
+    
+    this.state = {
+      qrUrl: '';
+    };
+  }
+
```

Comment:
Let's call this `updateUrl`? To make it clear this method has side effects. Generally `get` methods imply no side effects.